packages/client/lib/commands/CLUSTER_INFO.spec.ts

```diff
@@ -43,4 +44,11 @@ describe('CLUSTER INFO', () => {
             }
         );
     });
+
+    testUtils.testWithCluster('cluster.clusterInfo', async cluster => {
```

Comment:
Change the title to "clusterNode.clusterInfo" (not just here, in all the tests that uses `getSlotMaster`)